tests/ja3-lua-rules-quic/test-ja3.lua

```diff
@@ -0,0 +1,18 @@
+local ja3 = require("suricata.ja3")
+
+function init (args)
+   ja3.enable_ja3()
+   local needs = {}
+   needs["ja3"] = tostring(true)
```

Comment:
@jasonish is this ok ?

As we cannot use a hook since ja3 is in both TLS and QUIC protocols, I need to convey the information from lua to suricata that I want a ja3 and ja3 util lib functions do not have access to the current rule, so I resort to `init` return used by `DetectLuaSetup` (and which has access to the rule)